apps/sms/test/marionette/mocks/mock_split_view_mode.js

```diff
@@ -0,0 +1,14 @@
+/* global Services */
+'use strict';
+
+Services.obs.addObserver(function(document) {
+  if (!document || !document.location ||
+      document.location.origin !== 'app://sms.gaiamobile.org') {
+    return;
+  }
+
+  // Switch to split-view mode Inbox instead.
+  if(document.location.pathname === '/index.html') {
```

Comment:
wondering if we could add hidden entry points to our manifest, or just replace the manifest on the fly before generating the profile and launching mulet... I'm sure this was done in other apps BTW, I think I heard of it. Maybe ask Johan ? But let's not wait for this to move forward, please just file a separate bug.